web_advanced_search/__manifest__.py

```diff
@@ -5,9 +5,10 @@
 
 {
     "name": "Advanced search",
-    "version": "11.0.1.0.0",
+    "version": "11.0.1.0.2",
     "author": "Therp BV, "
               "Tecnativa, "
+              "Camptocamp, "
```

Comment:
I think these changes don't promote Camptocamp to co-author.

Reply:
fixed